docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,10 +1,19 @@
 = {ProductName}
+:toc: left
+:toclevels: 1
+:icons: font
+:numbered:
+:source-highlighter: highlightjs
 
-{ProductName} is a fully managed service that provides a simple workflow for developing, testing, and releasing containerized applications that meets enterprise security standards.
+Welcome to the {ProductName} documentation!
+
+== Why use {ProductName}?
+
+You can use {ProductName} to enable continuous integration and continuous development (CI/CD) for your applications in a matter of minutes. You can also use {ProductName} to secure your software supply chain by reaching link:https://slsa.dev/spec/v0.1/levels#summary-of-levels[SLSA level 3] security standards. And you can customize how {ProductName} builds, tests, and deploys containerized applications.  
```

Comment:
It might flow better if one of the sentences is a little longer here. Right now, all 3 sentences are about the same length, so it sounds a bit choppy